azure-devops/azext_devops/dev/team/arguments.py

```diff
@@ -68,6 +68,14 @@ def load_team_arguments(self, _):
         context.argument('yes', options_list=['--yes', '-y'], action='store_true',
                          help='Do not prompt for confirmation.')
 
+    with self.argument_context('devops project list') as context:
+        context.argument('state_filter', arg_type=get_enum_type(_PROJECT_GET_STATE_VALUE_FILTER),
+                         help='State filter.')
+        context.argument('continuation_token',
+                         help='Continuation token.')
+        context.argument('get_default_team_image_url', arg_type=get_three_state_flag(),
```

Comment:
@geverghe  Thoughts? Suggestions on a short name if you think we should add?

Reply:
Hmm...I don't have any ideas around a shorter name. However, is the help going to be only continuation token? 

For example, az storage queue list has a continuation token marker and the details are as follows:
--marker
An opaque continuation token. This value can be retrieved from the next_marker field of a previous generator object if num_results was specified and that generator has finished enumerating results. If specified, this generator will begin returning results from the point where the previous generator stopped.

Can we do this level of detail:
1. What does the continuation token do?
2. From where do you get it ?

Reply:
I meant for this one: **--get-default-team-image-url**

Firstly do we even need this option? 
If yes, is **--image-url** sufficient as a secondary short name.

--continuation-token seems still ok in length.

Reply:
Ah ok, got it. I agree with the above feedback. It looks to be a highly unlikely scenario that a project list may be done on team image - let me know if there is a scenario. I have not seen any request for same. Due to the same reason, i do not have any data point for it not to be included as well. I would recommend the former - not including it till we have clarity. In case it is added, instead of maybe --image-url, I would recommend expansion to --team-image-url to indicate the the entity to which the image url belongs to.